src/aat/resources/features/common/Base_Expecting_500.td.json

```diff
@@ -0,0 +1,10 @@
+{
+	"_guid_": "Base_Expecting_500",
+	"_extends_": "Base-Root",
+	"request": {
+	},
+	"expectedResponse": {
+		"responseCode": 500,
+		"responseMessage": "Internal Server Error"
+	}
+}
```

Comment:
There shouldn't be any base file under this branch. Such changes must be made to 6329. Besides, since you shouldn't expect 500, there shouldn't be a reference to this file already.

Reply:
Yes, base file 500 is deleted.